projects/grafana/grafana-ocp.yaml

```diff
@@ -0,0 +1,717 @@
+---
```

Comment:
For ease of re-deploy and update, we've started to break these up into:
 - config
 - deployment
 - build

Reply:
This comment is still outstanding. We need three separate yaml files with the pieces outlined by @stevekuznetsov